src/js/facility-locator/components/Listbox.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { truncate, kebabCase } from 'lodash';
```

Comment:
Can you switch this to the `import _ from 'lodash';` syntax? I'm not sure the lodash babel plugin picks this up to optimize.